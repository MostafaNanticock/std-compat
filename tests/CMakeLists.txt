include_directories(common)

add_subdirectory(any)
add_subdirectory(expected)
add_subdirectory(filesystem)
add_subdirectory(optional)
add_subdirectory(span)
add_subdirectory(variant)
