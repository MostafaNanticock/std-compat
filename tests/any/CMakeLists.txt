set(TEST_DEPENDENCIES
    StdCompat::any
)

std_compat_add_test(tst_any
    ${TEST_DEPENDENCIES}
    tst_any.cpp
)
