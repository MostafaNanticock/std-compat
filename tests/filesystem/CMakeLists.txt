set(TEST_DEPENDENCIES
    StdCompat::filesystem
)

std_compat_add_test(tst_filesystem
    ${TEST_DEPENDENCIES}
    tst_filesystem.cpp
)
