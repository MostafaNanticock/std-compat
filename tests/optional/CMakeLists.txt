set(TEST_DEPENDENCIES
    StdCompat::optional
)

std_compat_add_test(tst_optional
    ${TEST_DEPENDENCIES}
    tst_optional.cpp
)
