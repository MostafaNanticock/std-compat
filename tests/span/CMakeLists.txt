set(TEST_DEPENDENCIES
    StdCompat::span
)

std_compat_add_test(tst_span
    ${TEST_DEPENDENCIES}
    tst_span.cpp
)
