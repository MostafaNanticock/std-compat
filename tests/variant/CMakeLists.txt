set(TEST_DEPENDENCIES
    StdCompat::variant
)

std_compat_add_test(tst_variant
    ${TEST_DEPENDENCIES}
    tst_variant.cpp
)
