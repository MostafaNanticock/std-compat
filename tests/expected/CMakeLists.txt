set(TEST_DEPENDENCIES
    StdCompat::expected
)

std_compat_add_test(tst_expected
    ${TEST_DEPENDENCIES}
    tst_expected.cpp
)
